src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(Color color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round(width * height);
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: "
+                + this.getClass().getSimpleName().toLowerCase() + ", "
+                + "area: " + this.calculateArea() + " sq. units, "
+                + "width: " + this.width + ", "
+                + "height: " + this.height + ", "
+                + "color: " + getColor().toString()
+                + '}';
```

Comment:
The `toString` method should ensure consistent punctuation. Consider adding a space before the closing curly brace to maintain consistency in the output format.